### ISSUE TITLE
About the training strategy?

### ISSUE THREAD

**Issue by BIGJUN777** [bobwan1995/PMFNet#22]
Thank you for making your code available.

Would you tell me how you train your model on both the HICO-DET and V-COCO dataset?
1) In HICO-DET dataset, the data are split into two subsets named training set and test set. Did you train your model with all the data in the training set without further splitting the training set into the validation set and the smaller training set?
2) As for V-COCO dataset, from the code, I think you train your model on 'vcoco_trainval' set. How did you determine the proper hyper-parameters(i.e learning rate, batch size) to alleviate the overfitting problem just based on the training loss on 'vcoco_trainval' set?  

Look forward to your reply. @bobwan1995 @desenzhou
Best regard.

**Comment by oby-yang** [comment]
Do you have code that uses the HICO-DET dataset？I need it！

**Comment by BIGJUN777** [comment]
These projects might help you: [HO-RCNN](https://github.com/ywchao/ho-rcnn), [No-Frills](https://github.com/BigRedT/no_frills_hoi_det) and [GPNN](https://github.com/SiyuanQi/gpnn). I remember the latter two have python scripts to extract the raw data in HICO-DET. 
BTW, do you have any idea about the question above?

**Comment by wanna-fly** [comment]
> Thank you for making your code available.
> 
> Would you tell me how you train your model on both the HICO-DET and V-COCO dataset?
> 
> 1. In HICO-DET dataset, the data are split into two subsets named training set and test set. Did you train your model with all the data in the training set without further splitting the training set into the validation set and the smaller training set?
> 2. As for V-COCO dataset, from the code, I think you train your model on 'vcoco_trainval' set. How did you determine the proper hyper-parameters(i.e learning rate, batch size) to alleviate the overfitting problem just based on the training loss on 'vcoco_trainval' set?
> 
> Look forward to your reply. @bobwan1995 @desenzhou
> Best regard.

Usually they train the model on val set to find the best hyper-parameters and then train it on the trainval set with these parameters to achieve the best performance on the test split.

**Comment by bobwan1995** [comment]
1. I just train it on training set on HICO-DET for with fixed iterations and directly test on test set.
2. Usually wetrain the model on train set to find the best hyper-parameters on val set, and finally train it on the trainval set with these parameters to achieve the best performance on the test split.

**Comment by BIGJUN777** [comment]
Thanks, Got it. @bobwan1995 @wanna-fly
